### PR TITLE
[WIP] I606 workspace projects

### PR DESCRIPTION
**Pull request by petra-sargent**
### What does this PR do?
This fixes grammar issues and style issues. 

### What issues does this PR fix or reference?
#606

### CODE REVIEW

**Comment by petra-sargent** [comment]
@rkratky @robin-owen @supriya-bharadwaj  @preeticp  Please review. Thanks!

File: src/main/pages/workspace-admin/projects.adoc [hunk 1]
```diff
@@ -8,10 +8,12 @@ folder: workspace-admin
...
 
 [id="projects-in-workspaces"]
-== Projects in Workspaces
+== Projects in workspaces
 
 
-Projects are always associated with a workspace and saved in a workspace configuration:
+Projects are always associated with a workspace and saved in a workspace configuration.
+
+The following is an example of the YAML file:
```
Comment (kowen-rh) [inline src/main/pages/workspace-admin/projects.adoc]: What YAML file is this?

File: src/main/pages/workspace-admin/projects.adoc [hunk 2]
```diff
@@ -34,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
...
 
-image::workspaces/projects.png[]
+. Git projects 
+. remotely hosted archives
+. GitHub projects 
+. example projects provided by Eclipse Che 
 
-[id="sub-projects"]
-== Sub-Projects
+Project import tools can be found both on the *User Dashboard* when you are creating a new workspace or editing an existing one on the IDE or on the *Workspace* menu.
```
Comment (kowen-rh) [inline src/main/pages/workspace-admin/projects.adoc]: This is a long sentence with an awkward structure. Can we break that down some and simplify it?

File: src/main/pages/workspace-admin/projects.adoc [hunk 3]
```diff
@@ -34,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
...
+
+[id="creating-a-sub-projects"]
+== Creating a sub-project
+
+A sub-project is a portion of a project that can have sets of commands run against it where the sub-directory is treated as the root working directory. Sub-projects make it possible to organize a single repository into multiple, independently buildable, and runnable units.
+
+ To create a module, right click on a directory in the IDE explorer tree and select *Convert to Project*. You can then execute commands directly against this sub-projects.
+
+[id="navigating-the-project-tree"]
+== Navigating the project tree
```
Comment (kowen-rh) [inline src/main/pages/workspace-admin/projects.adoc:73]: :+1:

File: src/main/pages/workspace-admin/projects.adoc [hunk 4]
```diff
@@ -34,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
...
-2.  *Project types describe different aspects of a project* such as types of source files inside, the structure of the explorer tree, the way in which a command will be executed, associated workflows, and which plug-ins must be installed.
-3.  *A project defines a set of attributes.* The attributes of a project can be mandatory or optional. Attributes that are optional can be dynamically set at runtime or during configuration.
-4.  *Sub-projects may have different project types than their parents.* Modules may physically exist within the tree structure of the parent (as its subfolders) or outside (the parent is a soft link to the module project).
+1.  A project has a type.  The project type is defined as one primary type and zero or more mixin types. A primary project type is one where the project is editable, buildable, and runnable. A mixin project type defines additional restrictions and behaviors of the project, but by itself cannot be a primary project type. The collection of primary and mixin types for a single project define the aggregate set of attributes that will be stored as meta data within the project.
```
Comment (kowen-rh) [inline src/main/pages/workspace-admin/projects.adoc]: There's no need to explicitly number list items in AsciiDoc.

File: src/main/pages/workspace-admin/projects.adoc [hunk 5]
```diff
@@ -34,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
...
+2.  Project types describe different aspects of a project such as types of source files inside, the structure of the explorer tree, the way in which a command will be executed, associated workflows, and which plug-ins must be installed.
+3.  A project defines a set of attributes. The attributes of a project can be mandatory or optional. Attributes that are optional can be dynamically set at runtime or during configuration.
+4.  Sub-projects may have different project types than their parents. Modules may physically exist within the tree structure of the parent. For example, subfolders exists within the tree structure of the parent.   Also, modules may exist outside such as when the parent is a soft link to the module project.
```
Comment (kowen-rh) [inline src/main/pages/workspace-admin/projects.adoc]: "For example, subdirectories exist within the tree structure of the parent."

Also, there is excess whitespace (three spaces?) before "Also, modules may".

By "outside", do you mean outside of the tree structure? I'm not sure what this means.

File: src/main/pages/workspace-admin/projects.adoc [hunk 6]
```diff
@@ -34,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
...
 ----
 
-Once a project is saved into a workspace config, the IDE will check if the project exists on a file system, and if not, import it using `source.location` URL. It happens at IDE initialization stage.
+Once a project is saved into a workspace configuration, the IDE will check if the project exists on a file system.  If the project does not exist on the file sytem, you can import it using the `source.location` URL. This happens during the IDE initialization stage.
 
-You can add Git projects, remotely hosted archives, your GitHub projects or project samples that Eclipse Che offers out of the box. Project import tools can be found both in User Dashboard (when creating a new workspace or editing an existing one) on in the IDE, Workspace menu.
+You can add the following projects:
 
-image::workspaces/projects.png[]
+. Git projects
```
Comment (rkratky) [inline src/main/pages/workspace-admin/projects.adoc]: Why numbered list?

File: src/main/pages/workspace-admin/projects.adoc [hunk 7]
```diff
@@ -34,28 +36,40 @@ Projects are always associated with a workspace and saved in a workspace configu
...
-Once a project is saved into a workspace config, the IDE will check if the project exists on a file system, and if not, import it using `source.location` URL. It happens at IDE initialization stage.
+Once a project is saved into a workspace configuration, the IDE will check if the project exists on a file system.  If the project does not exist on the file sytem, you can import it using the `source.location` URL. This happens during the IDE initialization stage.
 
-You can add Git projects, remotely hosted archives, your GitHub projects or project samples that Eclipse Che offers out of the box. Project import tools can be found both in User Dashboard (when creating a new workspace or editing an existing one) on in the IDE, Workspace menu.
+You can add the following projects:
 
-image::workspaces/projects.png[]
+. Git projects 
+. remotely hosted archives
+. GitHub projects
```
Comment (rkratky) [inline src/main/pages/workspace-admin/projects.adoc]: What's the diff. between Git and GitHub projects?

**Review by rkratky — CHANGES_REQUESTED** [review]

**Comment by rkratky** [comment]
@petra-sargent, the conflict in this PR is caused by the fact that all Che 6 docs are now under their own umbrella directory in `src/main/pages/che-6/`. To resolve the conflict (after incorporating review changes), just move your file to the new structure.

File: src/main/pages/workspace-admin/projects.adoc (hunk 6 above)
Comment (petra-sargent) [reply src/main/pages/workspace-admin/projects.adoc]: doesn't need a numbered list

File: src/main/pages/workspace-admin/projects.adoc (hunk 7 above)
Comment (petra-sargent) [reply src/main/pages/workspace-admin/projects.adoc]: I am not sure what that means here. but could it mean that Git projects are projects on your local machine and GitHub projects are projects stored on the cloud.

Comment (kowen-rh) [reply src/main/pages/workspace-admin/projects.adoc]: Well, the idea here is that GitHub projects *are* Git projects. Is there some special support for GitHub versus any other Git hosting (GitLab, Bitbucket, a custom Git server, or local repositories)?

Comment (petra-sargent) [reply src/main/pages/workspace-admin/projects.adoc]: I'll open an issue to ask your question!

**Comment by petra-sargent** [comment]
I am closing this PR and opening PR #614 to handle this request. It is easier to open up a new PR to fix the merge conflict.

**Pull request closed without merging**